problem_builder/tests/integration/test_step_builder.py

```diff
@@ -624,9 +624,7 @@ def test_review_tips(self):
 
     @data(True, False)
     def test_conditional_messages(self, include_messages):
-        """
-        Test that conditional messages in the review step are visible or not, as appropriate.
-        """
+        # Test that conditional messages in the review step are visible or not, as appropriate.
         max_attempts = 3
         extended_feedback = False
         params = {
```

Comment:
@bradenmacdonald  

Tim asked me whether it's possible to add regression tests to this feature, and that is not easy for the following two reasons: 

1. Testing whether video is playing is non trivial, especially for videos embedded in HTML. 
2. Testing whether ``Html`` is detached is hard, as the workbench doesn't know how to handle ``html`` XModule, but rather uses ``html_demo`` xblock. For now only a ``html`` Xmodule can be added to step builder using studio interface. As of today I don't know how to fix this issue. 

So would it be sensible to push this without integration tests? 

CC: @itsjeyd